templates/default/config/siteConfig.js

```diff
@@ -21,23 +23,28 @@ const defaultConfig = {
   ],
   // optional additional nextSeo content set on each page
   // see https://github.com/garmeeh/next-seo
-//  nextSeo: {
-//    openGraph: {
-//      images: [
-//        {
-//          url: 'https://image.url/...',
-//          alt: '',
-//          width: 1200,
-//          height: 627,
-//          type: 'image/png',
-//        }
-//      ]
-//    }
-//  }
+  nextSeo: {
+    openGraph: {
+      images: [
+        {
+          url: userConfig.socialImageUrl 
+            ? userConfig.socialImageUrl 
+            : 'https://flowershow.app/assets/images/frontpage-screenshot.jpg',
```

Comment:
I don't think we should default this like this for all users. This should be empty by default.